### PR TITLE
refactor: rebrand switch

### PR DESCRIPTION
**Pull request by anuraghazra**

### CODE REVIEW

File: packages/blade/src/components/Switch/switchTokens.ts [hunk 1]
```diff
@@ -106,15 +91,15 @@ const switchMotion: Record<string, Record<string, `motion.${EasingString | Durat
...
   },
-};
+} as const;
 
-const switchHoverTokens: SelectorInputHoverTokens = {
+const switchHoverTokens = {
   default: {
     background: {
       checked: 'colors.brand.primary.600',
       unchecked: 'colors.brand.gray.600.lowContrast',
```
Comment (saurabhdaware) [inline packages/blade/src/components/Switch/switchTokens.ts:100]: we don't have these tokens anymore no 🤔

Comment (anuraghazra) [reply packages/blade/src/components/Switch/switchTokens.ts]: Oops, good catch. Forgot to make the hover tokens typesafe in SelectorInput and missed this.

**Review by saurabhdaware — APPROVED** [review]

**Review by snitin315 — CHANGES_REQUESTED** [review]
Need to export Switch.

https://github.com/razorpay/blade/blob/rebranding/master/packages/blade/src/components/index.ts

**Comment by snitin315** [comment]
@anuraghazra Build is failing, could you please check once?

**Comment by kamleshchandnani** [comment]
rebase

**Review by kamleshchandnani — APPROVED** [review]

**Pull request merged**